pkg/controller/controller_test.go

```diff
@@ -3444,6 +3597,55 @@ func TestProvisionFromPVC(t *testing.T) {
 			},
 		}
 
+		nilModePV := &v1.PersistentVolume{
```

Comment:
`nilModePV` is the same as `pv` above, and `pv` can be reused instead.

Reply:
Yeah, I'll go through these again and verify the modes again; thanks.

Reply:
Resolved by the latest update